### ISSUE TITLE
[CI] FullClusterRestartIT.testRandomDocumentsAndSnapshot fails

### ISSUE THREAD

**Issue by cbuescher** [elastic/elasticsearch#25111]
Failure happened on a PR build, but it reproduces on master: 

```
FAILURE 0.92s | FullClusterRestartIT.testRandomDocumentsAndSnapshot <<< FAILURES!
   > Throwable #1: java.lang.AssertionError: expected version to be one of [7.0.0,6.5.1] but was p 0 testrandomdocumentsandsnapshot_data 6.6.0
   > 	at __randomizedtesting.SeedInfo.seed([CC2082F088A3F238:4EC6C89239D544F4]:0)
   > 	at org.elasticsearch.upgrades.FullClusterRestartIT.assertTranslogRecoveryStatistics(FullClusterRestartIT.java:319)
   > 	at org.elasticsearch.upgrades.FullClusterRestartIT.testRandomDocumentsAndSnapshot(FullClusterRestartIT.java:228)
   > 	at java.lang.Thread.run(Thread.java:748)
```

This reproduces for me on current master:

gradle :qa:full-cluster-restart:v5.6.0-SNAPSHOT#upgradedClusterTestRunner -Dtests.seed=CC2082F088A3F238 -Dtests.class=org.elasticsearch.upgrades.FullClusterRestartIT -Dtests.method="testRandomDocumentsAndSnapshot" -Dtests.security.manager=true -Dtests.locale=ar-TN -Dtests.timezone=America/La_Paz

**Comment by jasontedor** [comment]
It means a Lucene version in a version constant is wrong.

**Comment by cbuescher** [comment]
@jasontedor thanks, thats what @nik9000 already suggested, changing the version works. I just opened a one-liner PR, but I can push that before CI is through if you think it is good.

**Comment by jasontedor** [comment]
I trust you, push away.

**Comment by cbuescher** [comment]
@jimczi already pushed a fix ;-) 68f1d4df5
